Documentation/api_rfc/v2_acl_api.md

```diff
@@ -0,0 +1,456 @@
+# v2 Auth and Security
+
+## etcd Resources 
+There are three types of resources in etcd
+
+1. user resources: users and roles in the user store
+2. key-value resources: key-value pairs in the key-value store
+3. settings resources: security settings, auth settings, and dynamic etcd cluster settings (election/heartbeat)
+
+### User Resources 
+
+#### Users
+A user is an identity to be authenticated. Each user can have multiple roles. The user have a capacity on the resource if one of the roles has that capacity.
```

Comment:
May want to use "capability" here (and in the rest of the doc), as it seems to be closer to the core meaning?